requirements.txt

```diff
@@ -1,6 +1,6 @@
-celery<5
+celery<=5.0.2
 gitpython
-kombu<5 # A celery dependency but it's directly imported
+kombu<=5.0.2 # A celery dependency but it's directly imported
```

Comment:
Should we just pin to `==5.0.2`?

Reply:
Should we pin it to an specific version?
`<=5.0.2` or `==5.0.2` will have our builds breaking as soon as there is a patch version bump in Fedora (which may happen at any time). Should we say we want it to be >=5 instead? We could include saying we want it <6 but at this point I am not sure if it would matter at all.

Reply:
In that case, I think >=5 would be the best.
On F33 is Python3.9 as a default version, I'll update CI.